pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/AvroSchema.java

```diff
@@ -102,12 +102,7 @@ public boolean supportSchemaVersioning() {
     }
 
     public static <T> AvroSchema<T> of(Class<T> pojo, Map<String, String> properties) {
-        ClassLoader pojoClassLoader = null;
-        if (pojo != null) {
-            pojoClassLoader = pojo.getClassLoader();
-        }
-        SchemaDefinition<T> schemaDefinition = SchemaDefinition.<T>builder().withPojo(pojo).withProperties(properties).build();
-        return new AvroSchema<>(parseSchemaInfo(schemaDefinition, SchemaType.AVRO), pojoClassLoader);
+        return AvroSchema.of(SchemaDefinition.<T>builder().withPojo(pojo).withProperties(properties).build());
```

Comment:
This change will change the behavior since we are using the pojo classloader before.

Reply:
The next call will perform the same behavior[Please take a look here](https://github.com/apache/pulsar/blob/5712695817c3ccace6047e65674061bb104a334f/pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/AvroSchema.java#L94)